index.html

```diff
@@ -535,6 +535,8 @@ <h2 class="CONNECT2"><a href="#">LET'S CONNECT</a> </h2>
                                   
         
                     <button class="submit0" type="submit"> Get in touch </button>
+                    
+                    <div class="error-message"> </div>
```

Comment:
- [ ] [OPTIONAL] Good job, you have done really well, I would suggest placing the error message on top of the submit button or below the email input field and also center it, currently, it is after the submit button and sometimes the error is not clear to the user